translationCoreApps/translationNotes_Check_plugin
# updated colors, removed old files

#### This pull request addresses:

updated colors in css, removed some very old subcomponent files that are no longer used



#### How to test this pull request:

open tNotes and make sure I didn't break it!


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/translationcoreapps/translationnotes_check_plugin/67)
